src/PIL/TiffImagePlugin.py

```diff
@@ -1281,11 +1291,11 @@ def _setup(self):
         if xres and yres:
             resunit = self.tag_v2.get(RESOLUTION_UNIT)
             if resunit == 2:  # dots per inch
-                self.info["dpi"] = int(xres + 0.5), int(yres + 0.5)
+                self.info["dpi"] = xres, yres
```

Comment:
Let's include tuple brackets for clarity


```suggestion
                self.info["dpi"] = (xres, yres)
```

(this was fun! https://twitter.com/dabeaz/status/1387085343468441600)